lms/templates/fields/field_dropdown.underscore

```diff
@@ -1,13 +1,11 @@
-<% if (editable !== 'never') { %>
-    <% if (title && titleVisible) { %>
-        <label class="u-field-title" for="u-field-select-<%- id %>">
-            <%- title %>
-        </label>
-    <% } else { %>
-        <label class="sr" for="u-field-select-<%- id %>">
-            <%- screenReaderTitle %>
-        </label>
-    <% } %>
+<% if (title && titleVisible) { %>
```

Comment:
I'm nervous about this change, because fields are used outside the learner profile. You might want to revert this, and just pass editable as never for your field.